eva/binder/statement_binder.py

```diff
@@ -129,6 +130,42 @@ def _bind_load_data_statement(self, node: LoadDataStatement):
 
         node.column_list = column_list
 
+    @bind.register(UploadStatement)
```

Comment:
If the upload binder is mostly reusing the code from the load binder, you can do something like this
```suggestion
    @bind.register(UploadStatement)
    @bind.register(LoadDataStatement)
```